src/classes/Constraints/ConstraintRow.js

```diff
@@ -27,6 +28,35 @@ Goblin.ConstraintRow.createConstraintRow = function() {
 	return row;
 };
 
+/**
+ * Resets row's jacobian to 0 effectively disabling the constrain for corresponding
+ * bodies.
+ *
+ * @param {boolean} nullify_a Whether to nullify first object's jacobian
+ * @param {boolean} nullify_b Whether to nullify second object's jacobian
+ */
+Goblin.ConstraintRow.prototype.nullify = function( nullify_a, nullify_b ) {
+	for ( var i = 0; i < 12; i++ ) {
```

Comment:
if (nullify_a) { for (var i = 0; i < 6; ++i) { this.jacobian[i] = 0; }}
if (nullify_b) { for (var i = 6; i < 12; ++i) { this.jacobian[i] = 0; }}

Reply:
Agree, fixed.